docs/docs/hydroflow_plus/quickstart/clusters.mdx

```diff
@@ -1,90 +1,70 @@
 
-```rust title="src/broadcast.rs"
-use hydroflow_plus::*;
+:::info
 
-pub struct Leader {}
-pub struct Workers {}
-
-pub fn broadcast(
-    flow: &FlowBuilder,
-) -> (Process<Leader>, Cluster<Workers>) {
-    let leader = flow.process();
-    let workers = flow.cluster();
-
-    // ...
-
-    (leader, workers)
-}
-```
+There are a variety of APIs for sending data to and reciving data from clusters. For example, we can `broadcast_bincode` to send copies to all members, or use the existing `send_bincode` if we have a custom algorithm to determine which cluster member should receive a piece of data.
 
-## Broadcasting Data
-When sending data between individual processes, we used the `send_bincode` operator. When sending data from a process to a cluster, we can use the `broadcast_bincode` operator instead.
-
-```rust
-let data = leader.source_iter(q!(0..10));
-data
-    .broadcast_bincode(&workers)
-    .for_each(q!(|n| println!("{}", n)));
-```
-
-The `Stream` returned by `broadcast_bincode` represents the data received on _each_ process in the cluster. Because all processes in a cluster run the exact same computation, we can then use the `for_each` operator directly on that stream to print the data on each process.
-
-## Deploying Graphs with Clusters
-To deploy this application, we must set up the Hydro Deploy configuration as before. Our deployment script (`examples/broadcast.rs`) instantiates multiple services for the leader process and the workers. Since this script defines the physical deployment, we explicitly instantiate multiple services for the cluster spec, returning a `Vec` of services. We also set a display name for each service so that we can tell them apart in the logs.
+:::
 
-```rust title="examples/broadcast.rs"
-use std::cell::RefCell;
+<CodeBlock language="rust" title="src/first_ten_cluster.rs">{getLines(firstTenClusterSrc, 7, 9)}</CodeBlock>
 
-use hydro_deploy::{Deployment, HydroflowCrate};
-use hydroflow_plus::deploy::TrybuildHost;
+On each cluster member, we will then do some work to transform the data (using `map`) and log out the transformed values locally (using `inspect`, which is useful for debugging logic).
 
-#[tokio::main]
-async fn main() {
-    let mut deployment = Deployment::new();
+<CodeBlock language="rust" title="src/first_ten_cluster.rs">{getLines(firstTenClusterSrc, 10, 11)}</CodeBlock>
 
-    let builder = hydroflow_plus::FlowBuilder::new();
-    let (leader, workers) = flow::broadcast::broadcast(&builder);
+Finally, we will send the data back to the leader. We achieve this using a variant of the APIs from before: `send_bincode_interleaved`. This is similar to `send_bincode` in that the elements are sent to the leader process, but the elements from different cluster members are mixed together into a single stream (regular `send_bincode` would emit tuples of cluster IDs and data).
```

Comment:
You say what type `send_bincode` emits, but you don't say what `send_bincode_interleaved` emits.